docs/snapshot.md

```diff
@@ -34,4 +34,4 @@ The snapshots are by default in `{LOG_LOCATION}\config` directory, where `{LOG_L
 
 **Note** : 
 1. Models passed in --models parameter while starting TorchServe are ignored if restoring from a snapshot.
-2. For windows, if shutdown snapshot file is not generated then you can use last snapshot file.
\ No newline at end of file
+2. For windows, if shutdown snapshot file is not generated then you can use last snapshot file.
```

Comment:
these look the same to me?

Reply:
I cannot find the configs in the directory, I believe the correct naming is config?

Reply:
Resolving. Probably a newline char diff